yolo/CMakeLists.txt

```diff
@@ -38,18 +38,21 @@ add_executable(YOLONode
 	src/yolo_node.cpp
 	src/YOLO.cpp
 )
+
 add_dependencies(YOLONode ${PROJECT_NAME}_gencpp)
+
 target_link_libraries(YOLONode
 	${catkin_LIBRARIES}
 	m
     pthread
     stdc++
 	cudart
-    cuda
+    cuda    
     cublas
 	curand
 )
 
+#change this to your darknet path!!!
```

Comment:
I don't think this comment is necessary any more after using the `${CMAKE_CURRENT_SOURCE_DIR} ` variable
```suggestion
```